core/webui-explorer/src/main/java/org/exoplatform/ecm/webui/component/explorer/rightclick/manager/DeleteManageComponent.java

```diff
@@ -401,22 +420,24 @@ private void processRemoveNode(String nodePath, Node node, Event<?> event, boole
       node.remove();
       parentNode.save();
     } catch (VersionException ve) {
+      LOG.warn("Error deleting node", ve);
       uiApp.addMessage(new ApplicationMessage("UIPopupMenu.msg.remove-verion-exception", null,
                                               ApplicationMessage.WARNING));
 
       uiExplorer.updateAjax(event);
       return;
     } catch (ReferentialIntegrityException ref) {
+      LOG.warn("Error deleting node", ref);

```

Comment:
Should it be LOG.error ?
```suggestion
      LOG.error("Error deleting node", ref);

```

Reply:
As discussed, will preserve the WARN logging level here and I will attempt to clarify more the difference between both levels in Best Practices.